### DISCUSSION TITLE
Edge improvement

### DISCUSSION THREAD

**Discussion by s-h-a-d-o-w** [danielgatis/rembg#494] [Ideas]
At the moment, at the edges of the subject, the background is replaced by a pretty dark gray, e.g.: https://raw.githubusercontent.com/danielgatis/rembg/master/examples/animal-3.out.png

It would be great if that was medium gray instead, to make edge treatment a bit easier.

But what would probably be best (would have to be tried with a variety of subjects) and is frequently done in VFX, is to:
- Clamp the alpha so that everything that was <255 becomes 0
- Using the input image and based on this alpha, repeat RGB at the edges of the subject outwards (until the area of wherever the original alpha was <255 but >0 is covered)
- Apply the alpha as it was before clamping

**Comment by TrungTri** [comment]
Hi all,

Is there anyway to mess with the config to make the edge of the subject better?

**Reply by s-h-a-d-o-w** [reply]
I would recommend having rembg only generate the alpha channel. Then you can even use the strategy I described.
